pandas/tests/arrays/test_numpy.py

```diff
@@ -154,6 +154,21 @@ def test_to_numpy():
     tm.assert_numpy_array_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    "data_content,data_type,expected_result",
+    [
+        ([1, 2, 3], pd.Float64Dtype(), np.array([1, 2, 3], dtype=np.float64)),
```

Comment:
Why not build the Series here instead? I think just two parameters `data` and `expected_results` is enough (I think `expected` is descriptive enough and more common, but no big deal).